scripts/e2e/config/kserve/overlay.yaml

```diff
@@ -31,49 +24,20 @@ metadata:
 data:
   agent: |-
     {
-        "image" : "kserve/agent:v0.9.0",
+        "image" : "kserve/agent:v0.11.0",
         "memoryRequest": "50Mi",
         "memoryLimit": "1Gi",
         "cpuRequest": "10m",
         "cpuLimit": "1"
     }
   batcher: |-
     {
-        "image" : "kserve/agent:v0.9.0",
+        "image" : "kserve/agent:v0.11.0",
         "memoryRequest": "50Mi",
         "memoryLimit": "1Gi",
         "cpuRequest": "10m",
         "cpuLimit": "1"
     }
-  credentials: |-
```

Comment:
Do we not need these specs anymore?

Reply:
We still need this, but we don't need to overwrite it via kustomize.